ecs-cluster/alerts/remedies/12007_REPLICATION_EVENT_RETRY_LIMIT_EXCEEDED.yaml

```
@@ -0,0 +1,5 @@
+- symptomid: OBJST-12007
+  description: Replication event exceeds retry limit.
+  remedies:
+    - Contact Dell service to find replication issue and do manual retrigger for that event.
```

no... we shouldn't be telling them to contact dell for this,  why can't they fix this themselves.  only has a last resort should they contact dell services.

```suggestion
    - Check the network connection between the setups to ensure they're not running into network problems.
    - Review the events for this issue to determine which ends of the replication is not working correctly
    - Retry the replication once the issues have been resolved.
```

---

Hi Eddie, this remedies is not for customer.

In CRR we have a side stream to collect events which current code have bug handling it. Once we know there is a bug handling this event (for example, event formatting does not match our protocol or some assumption not match), we will retry several times (some bug is a corner case which could pass after retry) and then put it in this side stream. To avoid infinate retry on this event and consume resource. 

This alert is raised only when an event is put in this side stream. This means the object will stays in pending state and no event will retrigger to do the replication, to notify OE or service team to take a look and find out why and put this event back to main stream. 

Please suggest if there is a proper way for this alert, if this kind of alert do not need remedies we can remove it.